src/pages/Login/login.js

```diff
@@ -0,0 +1,61 @@
+import Loader from '../../Components/Loader/Loader';
+import ImageComponent from './ImageComponent/ImageComponent'
+import Logo from './Logo/Logo';
+import CustomInput from './CustomInput/CustomInput'
+import { useState } from 'react';
+import CustomButton from './CustomButton/CustomButton';
+
+export default function Login() {
+  const [emailAddress, setEmailAddress] = useState('');
+  const [password, setPassword] = useState('');
+  const [error, setError] = useState('');
+  const isInvalid = password === '' || emailAddress === '';
+  const handleLogin = async (event) => {
+    event.preventDefault();
+
+    try {
+      
+    } catch (error) {
+      setEmailAddress('');
+      setPassword('');
+      setError(error.message);
+    }
+  };
+  const buttonProps = {
+    disabled: isInvalid,
+    type: 'submit',
+    styleButton: `bg-blue-medium text-white w-full rounded h-8 font-bold ${isInvalid && 'opacity-50'}`,
+  };
```

Comment:
I am not against it, but why have you created a dedicated constant for the button props only?
Why not for the custom input for example?
You should have one approach, either this or that to make your code look consistent.

Reply:
In the previous Pull request, I used Array Data for the Custom Input component, so I set all the properties to Array.

Reply:
As discussed, it will be part of a future PR